src/common/utility_unix.cpp

```diff
@@ -18,6 +18,7 @@
  */
 
 #include <QStandardPaths>
+#include <QtGlobal>
```

Comment:
You sure this is needed? I'd expect QtGlobal to be pulled (indirectly) by QStandardPaths already.

Reply:
This header is needed according to the Qt documentation. It might not be needed, but not including it means relying on undocumented behavior and might break in future updates.